### PR TITLE
Genome annot

### PR DESCRIPTION
**Pull request by polkhe**
formatted genome annotation tutorial+meta

### CODE REVIEW

File: Genome-Annotation/tutorials/general-introduction.md [hunk 1]
```diff
@@ -1,110 +1,140 @@
...
 Genome annotation is the process of attaching biological information to sequences.
 It consists of three main steps:
 
  - identifying portions of the genome that do not code for proteins
  - identifying elements on the genome, a process called gene prediction, and
  - attaching biological information to these elements.
 
-
-# 1. Introduction into File Formats
+> ### Agenda
```
Comment (bgruening) [inline Genome-Annotation/tutorials/general-introduction.md]: @polkhe are these `>` really needed?

File: Genome-Annotation/tutorials/general-introduction.md [hunk 2]
```diff
@@ -1,110 +1,154 @@
...
+# Structural Annotation
 
-# 2. Structural Annotation
-
-For the genome annotation we us a piece of the *Aspergillus fumigatus* [genome sequence](../input_data/Aspergillus_sequence.fasta) as input file. 
-
-
-
-## 2.1 Sequence Features
+> ### :nut_and_bolt: Comments
```
Comment (bebatut) [inline Genome-Annotation/tutorials/general-introduction.md]: No need for a box here. I think

File: Genome-Annotation/tutorials/general-introduction.md [hunk 3]
```diff
@@ -1,110 +1,154 @@
...
+> For the genome annotation we use a piece of the *Aspergillus fumigatus* [genome sequence](../input_data/Aspergillus_sequence.fasta) as input file. 
+{: .comment}
 
-First we want to get some general information about our sequence. Count the number of bases in your sequence (**compute sequence length**), and check for sequence composition and GC content (**geecee**). Plot the sequence composition as bar chart. 
+## Sequence Features
 
-<img src="../images/barchart_sequencecomposition.png" width="30%">
+> ### :pencil2: Hands-on: Sequence composition
+>
+> First we want to get some general information about our sequence.
```
Comment (bebatut) [inline Genome-Annotation/tutorials/general-introduction.md]: Can you move this sentence outside of the box? It is more an introduction to the hands-on

File: Genome-Annotation/tutorials/general-introduction.md [hunk 4]
```diff
@@ -1,110 +1,154 @@
...
 
-
-
-## 2.2 Gene Prediction
+## Gene Prediction
 
 At first you need to identify those structures of the genome which code for proteins. This step of annotation is called “structural annotation”. It contains the identification and location of open reading frames (ORFs), identification of gene structures and coding regions, and the location of regulatory motifs. Galaxy contains several tools for the structural annotation. Tools for gene prediction are **Augustus** (for eukaryotes and prokaryotes) and **glimmer3** (only for prokaryotes).
 
-We use **Augustus** for gene prediction. Use the genome sequence (FASTA file) as input. Choose the right *model organism*, *gff* format output, and select all possible output options.
+> ### :nut_and_bolt: Comment
```
Comment (bebatut) [inline Genome-Annotation/tutorials/general-introduction.md]: I think the box is not needed here

File: Genome-Annotation/tutorials/general-introduction.md [hunk 5]
```diff
@@ -1,110 +1,154 @@
...
+> We use **Augustus** for gene prediction.
+{: .comment}
+
+### Gene prediction
 
+:wrench: Use the genome sequence (FASTA file) as input. Choose the right *model organism*, *gff* format output, and select all possible output options.
```
Comment (bebatut) [inline Genome-Annotation/tutorials/general-introduction.md]: Hands-on box?

File: Genome-Annotation/tutorials/general-introduction.md [hunk 6]
```diff
@@ -1,110 +1,154 @@
...
+> <summary></summary>
+> </details>
+{: .question}
 
 ![augustus_output](../images/augustus_out.png)
 
+## tRNA and tmRNA Prediction
 
-## 2.3 tRNA and tmRNA Prediction
+Use **Aragorn** for tRNA and tmRNA prediction. As input file use the *Aspergillus* genome sequence. You can choose the genetic code (e.g. bacteria). Select the topology of your genome (circular or linear).  Are there tRNAs or tmRNAs in the sequence?
```
Comment (bebatut) [inline Genome-Annotation/tutorials/general-introduction.md]: It is an hands-on part here. I think

File: Genome-Annotation/tutorials/general-introduction.md [hunk 7]
```diff
@@ -1,110 +1,154 @@
...
 - identification of transmembrane domains in protein sequences
 - finding gene ontology terms
 - pathway information
 
+For similarity searches we use *NCBI BLAST+ blastp* to find similar proteins in a protein database.
 
-For similarity searches we use *NCBI BLAST+ blastp* to find similar proteins in a protein database. As input file, select the protein sequences from Augustus, choose the protein BLAST database *SwissProt* and the output format *xml*. 
+### Similarity search
 
+:wrench: As input file, select the protein sequences from Augustus, choose the protein BLAST database *SwissProt* and the output format *xml*.
```
Comment (bebatut) [inline Genome-Annotation/tutorials/general-introduction.md]: Hands-on box?

File: Genome-Annotation/tutorials/general-introduction.md [hunk 8]
```diff
@@ -1,110 +1,154 @@
...
+> <details>
+> <summary></summary>
+> </details>
+{: .question}
 
-Parsing the xml output (**Parse blast XML output**) results in changing the format style into tabular. What information do you see in the BLAST output?
-
-From BLAST search results we we want to get only the best hit for each protein. Therefore apply the tool **BLAST top hit descriptions** with *number of descriptions =1* on the xml output file. For how many proteins we do not get a BLAST hit? Choose the tool **Select lines that match an expression** and enter the following information: *Select lines from* [select the BLAST top hit descriptions result file]; *that* [not matching]; *the pattern* [gi]. 
+From BLAST search results we want to get only the best hit for each protein.
+:wrench: Therefore apply the tool **BLAST top hit descriptions** with *number of descriptions =1* on the xml output file.
```
Comment (bebatut) [inline Genome-Annotation/tutorials/general-introduction.md]: Hands-on box? Maybe combining all the previous ones in this section in a big hands-on box with question boxes inside. What do you think?

File: Genome-Annotation/tutorials/general-introduction.md (hunk 1 above)
Comment (polkhe) [inline Genome-Annotation/tutorials/general-introduction.md]: Thanks, made a box

**Comment by bgruening** [comment]
@bebatut yes, this tutorial needs some love. But this is a great step forward I think!
Ok to merge?

**Comment by bebatut** [comment]
Yes. I will merge

**Comment by bebatut** [comment]
Thank a lot @polkhe

**Comment by bgruening** [comment]
Thanks!

**Comment by bebatut** [comment]
Can you merge? I can not merge from the phone a PR with a failing Travis

**Comment by bebatut** [comment]
Thanks

**Pull request merged**